Apps/WebClient/src/ClientApp/src/utility/configUtil.ts

```diff
@@ -35,12 +35,15 @@ export default abstract class ConfigUtil {
 
     public static isServiceEnabled(serviceName: ServiceName) {
         const config = ConfigUtil.getFeatureConfiguration();
+        console.log("isServiceEnabled - config log:", config);
         if (config.services && config.services.enabled) {
-            return config.services.services.some(
+            const result = config.services.services.some(
                 (service) =>
                     service.name.toLowerCase() === serviceName.toLowerCase() &&
                     service.enabled
             );
+            console.log("isServiceEnabled: " + serviceName, result);
+            return result;
```

Comment:
Nit: I'm not fond of having log statements in places that may be called in each render, as that can clutter up the console.

Also, we generally use our ILogger (WinstonLogger) instead of console.log.

Reply:
sorry I'll remove this it was debugging with Cypress